geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/utils/EmptyScanner.scala

```diff
@@ -12,10 +12,13 @@ import java.io.IOException
 import java.util.Collections
 
 import org.apache.hadoop.hbase.client.{Result, ResultScanner}
+import org.apache.hadoop.hbase.client.metrics.ScanMetrics
 
 object EmptyScanner extends ResultScanner {
   override def next(): Result = throw new IOException("Next on an empty iterator")
   override def next(i: Int): Array[Result] = throw new IOException("Next on an empty iterator")
   override def close(): Unit = {}
   override def iterator(): java.util.Iterator[Result] = Collections.emptyIterator()
+  override def getScanMetrics(): ScanMetrics = null
+  override def renewLease(): Boolean = false
```

Comment:
these are breaking the build... possibly if you define them without `override` they would still work with 1.4, but might be best to just remove them

Reply:
Looks happy in both versions just removing the override.